src/main/java/graphql/language/FloatValue.java

```diff
@@ -40,6 +40,20 @@ public BigDecimal getValue() {
         return new ArrayList<>();
     }
 
+    @Override
+    public NodeChildrenContainer getNamedChildren() {
+        return NodeChildrenContainer.newNodeChildrenContainer().build();
+    }
+
+    @Override
+    public FloatValue withNewChildren(NodeChildrenContainer newChildren) {
+        if (!newChildren.isEmpty()) {
+            throw new IllegalArgumentException("Cannot pass non-empty newChildren to Node that doesn't hold children");
+        }
+
+        return this;
```

Comment:
This one does not seem to be finished.

Is this a BUG?

Reply:
well: you can't add children to a FloatValue, so we only allow empty children here

Reply:
I would create a helper method for these checks

```
@Override
    public FloatValue withNewChildren(NodeChildrenContainer newChildren) {
        NodeChildrenContainer.assertEmpty(newChildren);
        return this;
    }
```

Reply:
done via helper method